assets/app/views/_overview-deployment.html

```diff
@@ -29,11 +29,11 @@
       <div class="component-label">Deployment</div>
       <h3 class="h4">
         {{deploymentConfigId}}, #{{rc | annotation:'deploymentVersion'}}
-        <span class="pficon-layered" ng-if="deploymentConfig == null" data-toggle="tooltip" data-placement="right" title="Deployment config {{deploymentConfigId}} no longer exists" style="cursor: help;">
+        <span class="pficon-layered" ng-if="deploymentConfigMissing" data-toggle="tooltip" data-placement="right" title="Deployment config {{deploymentConfigId}} no longer exists" style="cursor: help;">
```

Comment:
want to kill `{{deploymentConfigId}}` while you're at it? Maybe `The deployment config this deployment was created from no longer exists.`?